reports/Treat2015Visulize.Rmd

```diff
@@ -0,0 +1,262 @@
+---
+title: "Treat peatlands 2015"
+author: "K Todd-Brown (ktoddbrown@gmail.com)"
+date: "8/3/2017"
+output:
+  pdf_document:
+    fig_caption: yes
+    toc: yes
+  html_document: default
+---
+
+```{r setup}
+library(SoilDataR) #library(devtools); install_github("ktoddbrown/soilDataR")
+library(ggplot2) #make pretty plots
+library(plyr) #data management
+library(dplyr)
+library(knitr)
+library(lubridate)
+library(tidyr)
+
+#mapping librarys to help with global/regional plots
+library(ggmap)
+library(maps)
+library(mapdata)
+library(fiftystater)
+```
+
+```{r loadData, cache=TRUE}
+Treat <- SoilDataR::processData_Treat2015(dir='../repoData/Treat_2015/')
+#ISCN3 <- SoilDataR::processData_ISCN3(dir='../repoData/ISCN_3/')
+```
+
+#Observation time
+```{r obsTime}
+location.df <- unique(Treat$field[, c('lat', 'long', 'observation_date', 
+                                       'state', 'country')])
+location.df$year <- location.df$observation_date
+
+location.df$obsDate <- ymd(paste(location.df$year, 1, 1, sep="-"))
+location.df <- location.df[order(location.df$lat, location.df$long, location.df$obsDate),]
+location.df$yrCut <- cut(location.df$year, c(1950, 1960, 1970, 1980, 1990, 2000, 2013), dig.lab=4)
+#location.df$yrCut <- location.df$year
+location.df$latCut <- cut(location.df$lat, seq(-90, 90, by=0.05))
+location.df$longCut <- cut(location.df$long, seq(-180, 180, by=0.05))
+
+timeSpaceCounts <- location.df %>%
+  mutate(inUSA = country %in% 'United States') %>%
+  group_by(yrCut, latCut, longCut, inUSA) %>%
+  tally
+```
+```{r plotTime}
+ggplot(location.df, aes(x=obsDate)) + geom_histogram()
+
+mapWorld <- borders("world", colour="gray80", fill="gray80") # create a layer of borders
+#ggplot() + mapWorld
+ggplot(location.df) +
+  mapWorld + 
+  #geom_hex(aes(x=long, y=lat), bins=200) + 
+  geom_point(aes(x=long, y=lat), alpha=0.1, color='red') +
+  scale_fill_gradient(trans='log10') +
+  theme_bw() +
+  theme(text=element_text(size=18)) +
+  labs(x='', y='') +
```

Comment:
Note this puts empty strings on the axis labels. To truly remove, use `+ theme(axis.title.x = element_blank())`.

Reply:
Done a65d8c465b06dc61eb3324e7082ae9b23b20c5a3